controllers/AuthController.js

```diff
@@ -285,11 +280,12 @@ export default class AuthController {
     if (process.env.NODE_ENV !== 'test') {
       sendResetEmail(req, email, token);
     }
-    return res.status(200).send({
-      email,
-      token,
-      message: 'Your reset link has been sent to your email'
-    });
+    // return res.status(200).send({
+    //   email,
+    //   token,
+    //   message: 'Your reset link has been sent to your email'
+    // });
```

Comment:
What did you leave this here?

Reply:
@Tunji17 👆🏽

Reply:
Sorry it was an oversight, i've removed it @mazma1